src/kestrel_datasource_stixshifter/config.py

```diff
@@ -171,4 +175,6 @@ def load_options():
         config = {"options": {}}
     if "fast_translate" not in config["options"]:
         config["options"]["fast_translate"] = FAST_TRANSLATE_CONNECTORS
+    if "translation_consumes" not in config["options"]:
+        config["options"]["translation_consumes"] = TRANSLATION_CONSUME_NUM
```

Comment:
Suggest to put the config:
```
options:
    async_translation_workers_count: 1
```
Basically rename `consumes` to `workers` since it may be producers in the future for the second queue, the preffix `async` describes the scope, which is not multi-processing.

We may also update `TRANSLATION_CONSUME_NUM` to `ASYNC_TRANSLATION_WORKERS_CNT`.